### ISSUE TITLE
bufferall focusing the wrong window/tab

### ISSUE THREAD

**Issue by Islidius** [cmcaine/tridactyl#819]
The ```bufferall``` command does not focus the right window.

Problem description:
Current window setup is 2 windows, first with 2 tabs and second one with 4 tabs.
I am currently in the second window tab 3 (at least this is what ```bufferall``` shows as my current location).
If I now type ```bufferall 1.2``` it switches to the second tab on my current window (```2.2```).
If I type ```bufferall 2.2``` it switches to the first window and the second tab.

It seems like the window indices are switched up.
One other thing that is strange is that the out of bound check is somehow also wrong.
If I assume that the indices are switched up ```1.3``` should go to the second window tab 3 but it goes to the first tab in the window I am currently in.

This problem seems to be mentioned in the pull request implementing ```bufferall``` [#690](https://github.com/cmcaine/tridactyl/pull/690). But could not be reproduced. As the pull request is already merged I am creating a new issue for this.

**Comment by glacambre** [comment]
Hi, thank you for reporting this issue!

I can't seem to reproduce it :/. What Firefox and Tridactyl version are you using? (you can know this by running `:version`). If you're not running  the latest version of the Tridactyl betas, could you try updating? (The bug you're experiencing might have been fixed in https://github.com/cmcaine/tridactyl/pull/771)

Can you try running `:jsb bufferall("1.2")` in the command line and tell us whether it works as expected?

Also, could you list the title of the tabs you're trying to switch to? This might be a completion autoselection issue, although I seriously doubt it.

**Comment by Islidius** [comment]
I was running on ```1.13.1``` (the latest stable version) and Firefox ```61.0.1 (64-Bit)```.
Running ```:jsb bufferall("1.2")``` did not work as it did the same as using ```bufferall``` normally.

Titles of the tabs do not seem to matter as I was using test search terms in google/duckduckgo.

I then updated to ```1.13.1-104-ge4fc98b``` and the same problem persisted.
Restarting Firefox after that fixed all problems.

Thinking back it might be that I did not completely restarted Firefox after installing the stable version.
So this could be a bug only appearing before the first restart. (that could the reason why you cannot reproduce it)
